packages/wallets/ONTO/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "@solana/wallet-adapter-ONTO",
+    "version": "0.4.0",
+    "author": "Solana Maintainers <maintainers@solana.foundation>",
+    "repository": "https://github.com/solana-labs/wallet-adapter",
+    "license": "Apache-2.0",
+    "type": "module",
+    "sideEffects": false,
+    "main": "lib/cjs/index.js",
+    "module": "lib/esm/index.js",
+    "types": "lib/esm/index.d.ts",
+    "exports": {
+        "import": "./lib/esm/index.js",
```

Comment:
The `module` and `import` fields should reference the .mjs files output by tsc-esm.

Reply:
have update

Reply:
Not sure what changed, this is still incorrect. Please check the package.json in the other wallet packages to see what to use for these fields.